bigtable/unit_tests/retry_server.go

```diff
@@ -0,0 +1,295 @@
+package main
```

Comment:
The go file doesn't have to be here, but the compiled go code is a server that we want to run system/integration tests against. Sorta like testing against the emulator. I was thinking we could have a public GCS bucket that contained the binaries for each platform and pull the appropriate one down during systest to run. Open to other ideas!